src/test/java/bankaccount/accounts.java

```diff
@@ -0,0 +1,90 @@
+package bankaccount;
+
+
+
+import java.util.Scanner;
+
+public class accounts {
+
+  int checkingAccountBalance;
+   int savingsAccountBalance;
+   int savingsAccountNum;
+   int checkingAccountNum;
+   private int accountNum = 1000;
+   int numAccount= 0;
+    Scanner scanner= new Scanner(System.in);
+
+    //method to deposit
+    public void deposit(){
+        System.out.println("PLease enter last 4digits account number you would like to deposit from");
+        int last_4= scanner.nextInt();
+        if (last_4==checkingAccountNum || last_4==savingsAccountNum) {
+            System.out.println("Enter the amount you would like to deposit");
+            double amount = scanner.nextDouble();
+            if (amount > 0) {
+                if (last_4 == checkingAccountNum) {
+                    checkingAccountBalance += amount;
+                } else if (last_4 == savingsAccountNum)
+                    checkingAccountBalance += amount;
+                System.out.println(amount + "$ has been added to balance");
+            } else System.out.println("please enter a correct amount you would like to deposit");
+        }
+    }
+
+    //method to withdraw
+    public void withdraw(){
+        System.out.println("Please enter last 4 digits account number you would like to withdraw from");
+        int last_4= scanner.nextInt();
+        if (last_4==checkingAccountNum || last_4==savingsAccountNum);{
+        System.out.println("Enter the amount you would like to withdraw");
+         double amount= scanner.nextDouble();
+        if (amount<=checkingAccountNum || amount<=savingsAccountNum){
```

Comment:
I believe you wanted to write savingsAccountBalance and checkingAccountBalance in this line.

Reply:
Right